lms/templates/instructor/instructor_dashboard_2/cohort-editor.underscore

```diff
@@ -53,9 +53,10 @@
                     </label>
                     <textarea name="cohort-management-group-add-students" id="cohort-management-group-add-students"
                               class="input cohort-management-group-add-students"
-                              placeholder="<%- gettext('e.g. johndoe@example.com, JaneDoe, joeydoe@example.com') %>"></textarea>
+                              placeholder="<%- gettext('e.g. johndoe@example.com, JaneDoe, joeydoe@example.com') %>"
+                              aria-describedby="email-notification-tip"></textarea>
```

Comment:
By using `aria-describedby` here, we can have assistive tech read the helper text that's underneath this form field, when focusing on this form field.